.github/workflows/main.yml

```diff
@@ -13,19 +13,19 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          python-version: 3.11.1
+          python-version: 3.11.2
       - uses: pre-commit/action@v2.0.3
 
   test-docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          python-version: 3.11.1
+          python-version: 3.11.2
```

Comment:
We should probably update this at the same time that we update the Pyodide version of Python to 3.11.2. Though it's not super important.

Reply:
It would be helpful if the pyodide build process used a `python-version` file which `actions/setup-Python` can also read.
* https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-file-input

Reply:
I think we would like to keep all build configs in a single file.

Reply:
Read the Python version from Makefile.envs